src/pages/list/List.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useEffect } from "react";
+import ListHeader from "./ListHeader";
+import RoomSortLine from "./RoomSortLine";
+import RoomLists from "./RoomLists";
+import ListFiltering from "./ListFiltering";
+import { getDistance } from "geolib";
+
+import * as S from "./List.styled";
+const List = () => {
+  const [roomList, setRoomList] = useState([]);
+  const [realList, setRealList] = useState();
+  const [isSelect, setIsSelect] = useState([true, false, false]);
```

Comment:
옵션이 수정될 때마다 이 state의 기본값 및 각 로직을 수정해야 할 것 같네요!
단일로 sorting하는 기능이기 때문에, 선택된 sort 옵션을 string으로 관리하는 selectedSort state로 구현하시면 될 것 같습니다.
```suggestion
  const [selectedSort, setSelectedSort] = useState("거리순");
```